src/Core/Components/InputFile/FluentInputFile.razor.cs

```diff
@@ -42,6 +42,14 @@ public FluentInputFile()
         .AddStyle("display", "none", () => !DragDropZoneVisible)
         .Build();
 
+    internal Dictionary<string, object?> Attributes
```

Comment:
Why not to set `disabled="@Disabled"` directly in the **div** tag?

Reply:
Done.